app/uk/gov/hmrc/onestopshopregistration/models/BusinessContactDetails.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.onestopshopregistration.models
+
+import play.api.libs.json._
+
+case class BusinessContactDetails (
+  fullName: String,
+  telephoneNumber: String,
+  emailAddress: String,
+  websiteAddress: String
```

Comment:
The frontend no longer has websiteAddress on this model, since websites are now collected separately as a list

Reply:
Removed websites from here